### PR TITLE
MAINT: update dependencies and versions

### PR DESCRIPTION
**Pull request by bsipocz**
This is to close https://github.com/fornax-navo/fornax-demo-notebooks/issues/184 and to address the comment in https://github.com/fornax-navo/fornax-demo-notebooks/issues/173#issuecomment-1850792665

### CODE REVIEW

File: light_curves/requirements-lc_classifier.txt
Comment (troyraen) [inline light_curves/requirements-lc_classifier.txt:1]: Starting from a clean conda environment, I needed the following to be added before I could import everything used in lc_classifier.md:

- `acstools`
- `requests`

**Review by troyraen — COMMENTED** [review]
A couple comments below. Otherwise, LGTM.

File: light_curves/requirements.txt [hunk 1]
```diff
@@ -3,15 +3,20 @@
 # the container supporting the science use case in question.
 acstools
 astropy
-astroquery==0.4.7dev8479 # pinned for Gaia bug astropy/astroquery#2844
+astroquery>=0.4.7.dev0
```
[the remark this answers is not in the document]
Comment (bsipocz) [reply light_curves/requirements.txt:6]: yes, it won't upgrade, but if you start from a new install, it'll pick up a good version. Given that the bug is in a section of dev versions but not before or after, this still feels a good way to state that.

File: light_curves/requirements-lc_classifier.txt
Comment (bsipocz) [reply light_curves/requirements-lc_classifier.txt:1]: ahh, good. I only grepped the import from the md file, but not from the code parts that it also run.

That being said, I think we should maybe limit the usage of requests, mast and panstars should ideally be accessible with astroquery/pyvo.

**Comment by bsipocz** [comment]
The review has been addressed, so I went ahead and merged this.

**Pull request merged**